File: microservice-frontApp/src/app/components/room_services/room.ts
export class Room {
  id!: any;
  roomNumber!: any;
  roomType!: RoomType;
  description!: any;
  surface!: any;
  price!: any;
}

export enum RoomType {

  SINGLE = 'SINGLE',
  DOUBLE = 'DOUBLE',
  TRIPLE = 'TRIPLE',
  STUDIO = 'STUDIO'
}

File: backend-microservices/booking-service/src/main/java/com/pi/bookingservice/dto/Room.java
package com.pi.bookingservice.dto;

import lombok.Getter;
import lombok.Setter;

import java.time.LocalDate;
@Getter
@Setter
public class Room {
    private String id;
    private String roomNumber;
    private String roomType;
    private String description;
    private Integer surface;
    private Double price;
}
